src/scripts/dungeons/DungeonBattle.ts

```diff
@@ -9,6 +9,7 @@ class DungeonBattle extends Battle {
         player.gainExp(this.enemyPokemon().exp, this.enemyPokemon().level, false);
         player.gainShards(this.enemyPokemon());
         player.addRouteKill();
```

Comment:
Should this be removed then?

Reply:
Yeah, looks like `player.route()` is always `0` when you're in a dungeon.
So it's just tracking total dungeon kills.